client/lib/user/gravatar.ts

```diff
@@ -0,0 +1,21 @@
+export type GravatarOptions = Partial< {
+	s: number;
+	d: DefaultImage;
+	f?: 'y';
+	forceDefault?: 'y';
+	r?: Rating;
+	rating?: Rating;
+} >;
+
+type DefaultImage =
+	| '404'
+	| 'mp'
+	| 'identicon'
+	| 'monsterid'
+	| 'wavatar'
+	| 'retro'
+	| 'robohash'
+	| 'blank'
+	| URL;
+
+type Rating = 'g' | 'pg' | 'r' | 'x';
```

Comment:
similar to above…this might go well in `client/gravatar/types.ts`

Reply:
ok yep I'll move it to `client/gravatar/types.ts`